django_mongodb/compiler.py

```diff
@@ -581,7 +613,7 @@ def _get_ordering(self):
         return tuple(fields), sort_ordering, tuple(extra_fields)
 
     def get_where(self):
-        return self.where
+        return getattr(self, "where", self.query.where)
```

Comment:
So we are back to `getattr()` for this... maybe we should drop the `get_where()` hook we previously added to try to avoid it. Those could be done in a follow up though. For now, perhaps a comment is warranted.

Reply:
okey.